app/qml/misc/GpsDataPage.qml

```diff
@@ -35,6 +34,13 @@ Page {
     }
   }
 
+  MapPosition {
+    id: mapPositioning
+
+    positionKit: __positionKit
```

Comment:
maybe we can use singleton and not pass it around?

Reply:
You mean to convert `MapPosition` to singleton too?

Reply:
`MapPosition` is a cpp class that needs mapSettings to work. MapSettings is not a singleton class so we would need to pass new map settings to it each time project is loaded

Reply:
No I mean why you need to have `positionKit: __positionKit` here and not use `__positionKit` directly in the element?

Reply:
Because it is a C++ class.

Reply:
make sense , sorry :)